mu-trees/addon/resolvers/glimmer-wrapper/index.js

```diff
@@ -103,9 +103,10 @@ const Resolver = GlobalsResolver.extend({
         source = `${type}:/${rootName}/`;
       } else if (source) {
         // make absolute
-        let parts = source.split(':src/ui/');
-        source = `${parts[0]}:/${rootName}/${parts[1]}`;
-        source = source.split('/template.hbs')[0];
+        // For apps:              template:src/ui/components/foo/template.hbs
+        // For addons: template:addon-name/src/ui/components/foo/template.hbs
+        let [, type, root = rootName, name] = /^([^:]+):(?:([^/]+)\/)?src\/ui\/(.+)(?:\/template\.hbs)$/.exec(source);
```

Comment:
Forgot to handle scoped packages.

```suggestion
        let [, type, root = rootName, name] = /^([^:]+):(?:((?:@[^/]+/)?[^/]+)\/)?src\/ui\/(.+)(?:\/template\.hbs)$/.exec(source);
```